server/service/orbit.go

```diff
@@ -450,3 +450,11 @@ func (svc *Service) SaveHostScriptResult(ctx context.Context, result *fleet.Host
 
 	return fleet.ErrMissingLicense
 }
+
+/////////////////////////////////////////////////////////////////////////////////
+// Post Orbit Disk Encryption result
+/////////////////////////////////////////////////////////////////////////////////
+
+type setOrUpdateDiskEncryptionResponse struct {
```

Comment:
When integrating with my PR for the orbit endpoint, this struct can be removed (the response struct for the endpoint is here: https://github.com/fleetdm/fleet/pull/14004/files#diff-6ae374180fab3c63484cf9f776cbd09ab34b425eea5fcd79395a23446b6c1a16R475)

Reply:
Done